lib/daemon_runner/client.rb

```diff
@@ -108,25 +126,60 @@ def parse_task(task)
     end
 
     # @private
+    # @param [Class] instance an instance of the task class
+    # @return [Hash<Symbol, String>] schedule parsed in parts: Schedule type and timing
+    def parse_schedule(instance)
+      valid_types = [:in, :at, :every, :interval, :cron]
+      out = {}
+      task_schedule = if instance.respond_to?(:schedule)
+        instance.send(:schedule)
+      else
+        schedule
+      end
+
+      raise ArgumentError, 'Not enough element in the Array' if task_schedule.length < 2
```

Comment:
What do this really mean though?  Maybe: "Malformed schedule definition, should be [TYPE, DURATION]"